pynicotine/gtkgui/search.py

```diff
@@ -556,20 +557,18 @@ def update_filter_comboboxes(self):
             widget.set_row_separator_func(lambda *_args: 0)
             widget.remove_all()
 
-            presets = self.FILTER_PRESETS.get(filter_id)
-
-            if presets:
-                for value in presets:
-                    widget.append_text(value)
-
-                widget.append_text("")  # Separator
-
             for value in config.sections["searches"][filter_id]:
                 widget.append_text(value)
 
+            presets = self.FILTER_PRESETS.get(filter_id)
```

Comment:
I'd prefer to have fewer presets instead of moving them to the bottom of the list.

Reply:
Nevermind, there are only 5 of them. I'd say that's fine to keep on the top.

Reply:
The advantage of having the most recently used at top is that it makes it much easier to toggle the filter on/off/on/off.

Not only is it easier to do for the user, but also easier to code such a function to act on the TextEntry icon handler.

Reply:
Not sure about this yet. It's something to explore for another PR.